### PR TITLE
Initial steps towards supporting distributed skeleton analysis

### PR DESCRIPTION
**Pull request by GenevieveBuckley**
This PR:
- [x] Remove degrees_image from `skeleton_to_csgraph()` function output
- [x] Adds a separate function to create 1degrees_image1
- [x] Replaces `skan.nputil.pad` with `numpy.pad`

### CODE REVIEW

**Comment by GenevieveBuckley** [comment]
The tests pass locally in my environment, for this PR branch.

File: doc/getting_started.rst [hunk 1]
```diff
@@ -506,9 +504,3 @@ This is of course a toy example. For the full dataset and analysis, see:
 But we hope this minimal example will serve for inspiration for your
 future analysis of skeleton images.
 
-If you are interested in how we used `numba <https://numba.pydata.org/>`_
-to accelerate some parts of Skan, check out `jni's talk <https://www.youtube.com/watch?v=0pUPNMglnaE>`_
-at the the SciPy 2019 conference.
-
-
-.. code-links:: python clear
```
Comment (jni) [inline doc/getting_started.rst:514]: Why did you remove this @GenevieveBuckley? Or is it just that you are on an older master branch?
